rio_tiler/io/cogeo.py

```diff
@@ -20,11 +20,11 @@
 from ..errors import DeprecationWarning, ExpressionMixingWarning, TileOutsideBounds
 from ..expression import apply_expression, parse_expression
 from ..utils import has_alpha_band, has_mask_band
-from .base import BaseReader
+from .base import BaseReader, SpatialMixin
 
 
 @attr.s
-class COGReader(BaseReader):
+class COGReader(BaseReader, SpatialMixin):
```

Comment:
Likewise isn't this not necessary because the mixin is in the `BaseReader` already?